### DISCUSSION TITLE
Coockie météo france

### DISCUSSION THREAD

**Discussion by nbaudine-Sol** [AmauriC/tarteaucitron.js#826] [Q&A]
Bonjour j'ai un abonnement pro et j'aimerais savoir si dans le futur le service "météo france" sera bientôt implémenter ? Mes sites l'utilise et j'avoue que cela m'embête un petit peu de n'avoir qu'une gestion partiel des cookies. Merci par avance ! 

Cordialement

**Comment by AmauriC** [comment]
Je peux sûrement l'ajouter mais il me faut le marqueur html de Météo France pour que je confirme si c'est faisable ou non ; )

**Reply by nbaudine-Sol** [reply]
Super merci à vous !

Le script JS:
`<script charset="UTF-8" src="http://www.meteofrance.com/mf3-rpc-portlet/rest/vignettepartenaire/172140/type/VILLE_FRANCE/size/PORTRAIT_VIGNETTE" type="text/javascript"></script>`


L'iframe :
`<iframe src="https://meteofrance.com/widget/prevision/172140" id="widget_meteo" scrolling="no" width="150" height="300" frameborder="0"></iframe>`


La balise : 
`<a class="link-article" target="_blank" href="https://meteofrance.com/previsions-meteo-france/luchat/17600">
`

Si vous avez besoin de plus d'infos n'hésitez pas.

**Comment by AmauriC** [comment]
A quoi correspond `172140` ?
`VILLE_FRANCE` c'est en toute lettre? Avec ou sans accent?
`PORTRAIT_VIGNETTE` c'est soit portrait, soit vignette?

Merci : )

**Reply by nbaudine-Sol** [reply]
Hmm en toute honnêté je ne sais pas vraiment, pour ajouter cette iframe je suis juste passé par le site de météo france https://meteofrance.com/widgets. Le code 172140 est générer de lui même dans l'iframe.

**Reply by AmauriC** [reply]
Ok, c'est le code INSEE de la ville, un peu différent du code postal.
J'ai tout ce qu'il faut, j'essai de l'ajouter rapidement ; )

**Reply by nbaudine-Sol** [reply]
Super ! Merci encore à vous

**Comment by nbaudine-Sol** [comment]
Ah et si d'une pierre de coup vous pourriez ajouter météo city ce serait parfait.
Voici le générateur de widget (c'est le même genre) : https://www.meteocity.com/widget/

**Comment by nbaudine-Sol** [comment]
Bonjour, suite à notre discussion, mon entreprise me demande vers quand ces 2 services seront implémenter. Auriez-vous une plage de dates potentielles à me donner que je pourrais leur communiquer ? 

Merci par avance !

**Reply by AmauriC** [reply]
J'essai de m'en occuper dans la semaine ; )

**Comment by AmauriC** [comment]
C'est dispo : )
https://github.com/AmauriC/tarteaucitron.js/commit/fcba3b094e7c0dca70b75d42b4dadc0ea6333e15
